crates/sui-framework/sources/governance/genesis.move

```diff
@@ -104,11 +103,11 @@ module sui::genesis {
                 primary_address,
                 worker_address,
                 // Initialize all validators with uniform stake taken from the subsidy fund.
-                balance::split(&mut subsidy_fund, initial_validator_stake_mist),
+                option::some(balance::split(&mut subsidy_fund, initial_validator_stake_mist)),
                 option::none(),
```

Comment:
Do we want to remove this lock thing while we're at it? Or should we just do that as a separate step?

Reply:
That will be a separate PR.